tomcat-manager/Dockerfile

```diff
@@ -9,6 +9,8 @@ USER root
 ADD https://github.com/Apiman/apiman/releases/download/$APIMAN_VERSION/apiman-distro-tomcat-$APIMAN_VERSION-overlay.zip $CATALINA_HOME
 RUN groupadd -r tomcat && useradd --no-log-init -r -g tomcat tomcat
 
+ADD https://jdbc.postgresql.org/download/postgresql-42.2.23.jar $CATALINA_HOME/lib
```

Comment:
Can you put that to the other ADDs.
Try to keep all adding at one place (makes it a little bit easier to read.